docs/ko/reference/overlay.md

```diff
@@ -91,7 +91,7 @@ function Example() {
   return (
     <button
       onClick={async () => {
-        const agreed = await overlay.openAsync<boolean>(({ isOpen, close }) => {
+          const agreed = await overlay.openAsync<boolean>(({ isOpen, close }) => {
           const agree = () => close(true);
```

Comment:
The `overlay.openAsync()` function accepts a callback function as its argument.
The content of the callback starts on line 95, so the indentation of lines 94 and 95 should differ accordingly.

Reply:
I misunderstood 😅 
I'll close this PR."